lib/dogapi/v1/monitor.rb

```diff
@@ -231,6 +231,33 @@ def get_all_downtimes(options = {})
         end
       end
 
+      #
+      # HOST MUTING
```

Comment:
Indeed; this endpoint is actually a convenience wrapper around downtime scheduling.